linux_os/guide/system/software/sudo/sudo_remove_no_authenticate/ansible/shared.yml

```
@@ -0,0 +1,7 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ ansible_sudo_remove_config("not authenticate", "!authenticate") }}}
```

```suggestion
{{{ ansible_sudo_remove_config("!authenticate", "!authenticate") }}}
```

I'd make all instances of "not authenticate" as "!authenticate". The use of quotes in the name tasks should make "!" a string vs something executable by shell.